### PR TITLE
added question php

### PR DESCRIPTION
**Pull request by haxkd**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [x] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [x] I have made small correction/improvements

### Changes / Instructions

_Add instructions to me, please type here, thanks_

### CODE REVIEW

**Review by Ebazhanov — APPROVED** [review]
looks beatiful! :hibiscus:

File: php/php-quiz.md [hunk 1]
```diff
@@ -1538,3 +1538,24 @@ echo "excellent";
...
+}
+else {
+    echo $calc;
+}
+```
+
+- [ ] 10000
+- [ ] 31
+- [ ] 1001
+- [ ] The moon is made of green cheese
```
Comment (chrishalbert) [inline php/php-quiz.md:1558]: I agree with your answer, however, you need to add an `x` between the brackets like ` - [x] The moon is made of green cheese`

Comment (Ebazhanov) [reply php/php-quiz.md:1558]: @haxkd can you pls take a look, thanks

Comment (Ebazhanov) [reply php/php-quiz.md:1558]: fyi i have quickly checked with chat gpt and the answer is = 31

Comment (haxkd) [reply php/php-quiz.md:1558]: @chrishalbert answer already marke by another PR.

@Ebazhanov answer is "The moon is made of green cheese" after running it into my local env.

**Pull request merged**